pkg/varnishcluster/controller/varnishcluster_grafana_dashboard_template.go

```diff
@@ -2342,20 +2341,26 @@ var grafanaDashboardTemplate =
       {
         "allValue": null,
         "current": {
+          "selected": false,
           "text": "All",
           "value": "$__all"
         },
-        "datasource": "{{.DatasourceName}}",
-        "definition": "label_values(varnish_up, instance)",
+        "datasource": "Prometheus-varnish",
+        "definition": "label_values(varnish_up{ service=\"{{.ServiceName}}\", namespace=\"{{.Namespace}}\", namespace=\"{{.Namespace}}\" }, pod)",
+        "description": null,
+        "error": null,
         "hide": 0,
         "includeAll": true,
-        "label": "Instance",
+        "label": "Pod",
         "multi": false,
-        "name": "varnish_instance",
+        "name": "varnish_pod",
         "options": [],
-        "query": "",
+        "query": {
+          "query": "label_values(varnish_up{ service=\"{{.ServiceName}}\", namespace=\"{{.Namespace}}\", namespace=\"{{.Namespace}}\" }, pod)",
+          "refId": "StandardVariableQuery"
+        },
         "refresh": 1,
-        "regex": "/^(.*):/",
+        "regex": "",
```

Comment:
Also what is the reason for removing the regex?

Reply:
So before I needed the IP address of the pod, so I queried the `instance` label that has the IP and the port (e.g. `10.0.0.12:1232` -> `10.0.0.12`). That regexp extracted the IP. Now I need only the pod name which comes without anything else.